ansible_module/conns/qubes.py

```diff
@@ -59,9 +59,7 @@
 
 # this _has to be_ named Connection
 class Connection(ConnectionBase):
-    """
-    This is a connection plugin for qubes: it uses qubes-run-vm binary to interact with the containers
-    """
+    """This is a connection plugin for qubes: it uses qubes-run-vm binary to interact with the containers."""
```

Comment:
Would be nice if you could specify the parameters too:

```
:param <type> play_context: 
```
this or Google Style
```
Args: 
    play_context: 
```